src/legacy/views/preview/contact-preview-header.test.jsx

```diff
@@ -31,7 +33,7 @@ describe.skip('Contact Preview Header', () => {
 			innerWidth: 1024,
 			innerHeight: 768
 		});
-		// const displayName = `${faker.name.firstName()} ${faker.name.lastName()}`;
+		// const displayName = `${faker.person.firstName()} ${faker.person.lastName()}`;
```

Comment:
Can we remove commented lines?

Reply:
I removed the whole file since it contains only a couple of simple tests, both skipped and not working